source/npmhub.js

```diff
@@ -56,6 +57,13 @@ async function init() {
       '.repository-content', // Old container https://github.com/refined-github/refined-github/issues/5751
     ]),
   });
+
+  const buttonTarget = document.querySelector(['.BorderGrid-cell [href=\'#readme\']']).parentNode;
+  const frag = document.createDocumentFragment();
+
+  new ScrollButton({target: frag});
+
+  buttonTarget.before(frag);
```

Comment:
I assume this moves the "Readme" link down. Let's just append the npmhub link at the end of that list instead of moving existing links.